airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_json_schema_helper.py

```diff
@@ -230,7 +230,7 @@ def test_get_object_strucutre(object, pathes):
     ],
 )
 def test_get_expected_schema_structure(schema, pathes):
-    assert get_expected_schema_structure(schema) == pathes
+    assert pathes == get_expected_schema_structure(schema)
```

Comment:
```suggestion
    assert paths == get_expected_schema_structure(schema)
```

nittiest of nits, but probably `paths`? technically unrelated to your change though

Reply:
fixed!